src/Metadata/TargetsMetadata.php

```diff
@@ -185,6 +185,9 @@ protected function getInfo(string $target): \ArrayObject
     public function getDelegatedKeys(): array
     {
         $keys = [];
+        if (!isset($this->getSigned()['delegations'])) {
+            return $keys;
+        }
         foreach ($this->getSigned()['delegations']['keys'] as $keyId => $keyInfo) {
```

Comment:
This could maybe be a little less verbose if we did this:

```php
foreach ($this->getSigned()['delegations']['keys'] ?? [] as $keyId => $keyInfo) {
```

What do you think?